release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -164,6 +164,14 @@ protected override void OnDestroy()
 		base.OnDestroy();
 	}
 
+	void OnApplicationFocus(bool hasFocus)
+	{
+		if (bNativeEditCreated)
+		{
+			this.SetVisible(hasFocus);
```

Comment:
If we resume the application this would try to focus all the input fields we have. I think we should only do this if `hasFocus` if false.